res/sections/25-Lez07032018.tex

```diff
@@ -17,134 +24,175 @@ \subsection{Analisi di mercato}
-per il progetto che si sta sviluppando\footnote{Si parla solitamente attorno ai
-10.000-30.000\euro{}, mentre in america si parla di 300.000-700.000\euro{}. La
-differenza, oltre d'investimento, è che gli incubatori in Italia richiedono una
-\textit{fee} mensile per il pagamento dei servizi, oltre ad una quota che si
-aggira per il 30-35\% della start-up. Gli americani, invece, non richiedono
-costi iniziali, ma cercano di guadagnare al momento della exit, ottenendo una
-precedenza sulla vendita delle quote societarie. Il fatto è che l'incubatore
-Italiano guadagna più tempo tu rimani al suo interno, quello americano il
-contrario: il guadagno avviene prima si esce. Nel primo caso non c'è coerenza
-negli obiettivi, nel secondo si.}. Incubatori con qualità più alta hanno
-selezioni d'accesso più dure, ma presentano una qualità delle start-up
-incubate. Questo atteggiamento indica anche una maggiore attenzione al fatto
-che le start-up che entrano siano di successo, ovvero si cerca di ottenere un
-\textbf{track record}\footnote{Storico delle \textbf{exit} di successo delle
-start-up di un incubatore} alto.
-
-Gli \textbf{FFF}\footnote{Ovvero i \textit{friendly, friends and fools},
-indicando rispettivamente i parenti/persone strette, gli amici e gli ``stolti''
-ovvero chi si fa convincere d'investire subito nell'azienda.} possono aiutare
-nella fase di bootstrap.\todo{Non ha detto nient'altro a riguardo!}
-
- \item a questo punto è possibile cominciare ad andare da investori seri per
-poter finire sul mercato. Si parte con attività di visibilità e marketing per
-mettere in risalto ciò che viene venduto. I \textbf{Business Angels} possono
-aiutare: essi sono un primo gruppo di investori, a cui poi seguono, nella fase
-successiva, i Venture Capital. Per le differenze tra i due
-vedesi~\ref{startup:crearestartup:bavsvc}. Quando entrano in gioco i business
-angel si dice che le quote dei founders si \textit{diluiscono}: il valore della
-start-up aumenta, anche se le quote di partecipazione dei fondatori
-diminuiscono. Questo, verso la fine, potrebbe portare al termine delle quote di
-maggioranza dal punto di vista di founders, anche se possono continuare a
-essere considerate azionisti di maggioranza.
+ nasce cioè la start-up. Fin prima, infatti, l'idea era solo su carta.
+ L'obiettivo è la creazione del \textbf{MVP}, ovvero il \textit{Minimum Viable
+ Product}, cioè un prodotto non completo con delle funzionalità chiave minime (
+ non è un prototipo, in quanto un prototipo è il primo prodotto completo).
+ L'MVP presenta caratteristiche minime in quanto permette la sua produzione in
+ tempi ristretti: un errore di molta start-up è voler uscire con un prodotto
+ troppo rifinito per essere un MVP, e ciò richiede troppo tempo per essere
+ fatto.
+ 
+ Lo scopo di un MVP è di ottenere feedback dal mercato e se ci sono clienti
+ interessati. Ciò è fondamentale in quanto approcciandosi ad aree nuove di
```

Comment:
Riscriverei questa frase con:
`Lo scopo di un MVP è di ottenere sia un feedback dal mercato sia un'avvicinamento di eventuali clienti interessati`

Però sta in piedi anche così, quindi sei libero di non cambiarla :grin: